backend/src/main/java/meetup/connect/event/EventServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package meetup.connect.event;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class EventServiceImpl implements EventService {
```

Comment:
suggestion: I don't think there's a need for a separate interface when you have only one implementation